test/1.2/operators/T_sizeof.dml

```diff
@@ -39,8 +39,8 @@ method test -> (bool ok) {
           (sizeoftype size_t == sizeoftype void *));
 
     // check that all sizeof invocations return a size_t
-    log "info": "%zd %zd %zd %zd", sizeof x, sizeof $x,
-        sizeoftype uint32, sizeoftype (t);
+    SIM_log_info(1, $dev.obj, 0, "%zd %zd %zd %zd", sizeof x, sizeof $x,
```

Comment:
What's this change about?

Reply:
Oh, I just found it on the way. IIRC, `log` adjust format strings to make qualifiers match args, so passing it as a `%zd` arg to `log` doesn't necessarily check size_t-ness, but passing it to SIM_log_info does. (format string conversions happen correctly in 1.4; I don't remember what limitations there are in 1.2 but this change is arguably an improvement)

Reply:
Alright.